Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -55,8 +54,9 @@ private string Me
 
         #region Grain overrides
 
-        public override Task OnActivateAsync()
+        public override async Task OnActivateAsync()
         {
+            await ReadStateAsync();
```

Comment:
I did not know it was done implicitly... It would not have mattered visibly for the sample since it's unlikely a grain will be deactivated in practice when you run it for a few minutes.  I'll take it out